kpi/views/browser_kpis.view.lkml

```diff
@@ -35,10 +59,8 @@ view: browser_kpis {
     hidden: yes
   }
 
-  measure: user_count_qualified {
```

Comment:
this is redundant with the unified_metrics.client_qualifies dimension. im removing to avoid confusion.